src/utils/getTestFrameworkName.ts

```diff
@@ -17,3 +28,13 @@ export function getTestFrameworkName(deps = {}) {
 
   return "N/A";
 }
+
+function isNodeTestRunnerUsed(scripts : Record<string, string>) : boolean {
+  const [majorVersion] = process.versions.node.split(".").map(Number);
+  if (majorVersion <= 18) {
+    return false;
+  }
```

Comment:
We don't need that, here you'r checking the Node.js version currently running by the dashlog process (not the Node.js version supported by the project we are scanning)